### ISSUE TITLE
Re-amalgamate in-house SQlite after update to 5.15.2

### ISSUE THREAD

**Issue by dmik** [bitwiseworks/qtwebengine-chromium-os2#38]
Configure and amalgamated versions were reset to upstream version due to many conflicts as it's easier to just redo this step.

Needed for https://github.com/bitwiseworks/qtwebengine-os2/issues/8 and the rest.

**Comment by dmik** [comment]
It appears that the new amalgamation script needs python3, sigh... Good news that @SilvanScherrer has it successfully ported just yet, so it might work. Will try.

**Comment by dryeo** [comment]
Paul also has a port that seems to work mostly. Needs pythonhome and 
pythonpath set.
http://smedley.id.au/tmp/python-3.8.5-os2-20200729.zip

**Comment by dmik** [comment]
@dryeo Thanks but no need in that; with some manual rpm workarounds, Silvan's RPMs worked great. They just need some small adjustments to be easily installable with yum (both python2 and python3 perfectly coexist in one installation) which will be added soon.

I've successfully reconfigured and re-amalgamated SQLite (and the Chromium startup problems went away). But I noticed some oddities in the upstream as well. In the previous version, the amalgamated sources were older than the in-house source tree they are expected to be made from (see https://github.com/bitwiseworks/qtwebengine-chromium-os2/issues/17#issuecomment-679335394). In this version, the amalgamated sources from upstream are NEWER than the in-house source tree... The former was version 3.32.1 while the latter is 3.31.1. Of course, re-amalgamation made the former version 3.31.1 too but I home it's fine for Chromium. I really wonder why the Chromium guys have such a mess there.
